egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,448 @@
+#!/bin/bash
+#
+# Copyright 2018  Nagendra Kumar Goel,
+#            Saikiran Valluri, Govivace.Inc -  Apache 2.0
+
+# The script is organized as below.
+# First we train the baseline LSTMP-TDNN config chain model for few epochs on the (Fisher+swbd)-english data,
+# Then, we perform SVD based refactoring of all the Affine components in this baseline final.mdl,
+# in order to reduce the overall model parameters size,
+# as determined by the bottleneck dim value or Energy and Shrinkage threshold values.
+# Then, we finetune the weight parameters of the refactored model using entire Fisher + switchboard data for single epoch.
+
+# Command used for comparing  WERs of decoding on different testsets using pre-SVD and SVD models:
+#  ./local/chain/compare_wer_general.sh --looped tdnn_lstm_1a_sp tdnn_lstm_1a_svd_sp
+#
+# Please run this entire script till the end before running the above WER compare command...
+
+
+# System                tdnn_lstm_1a_sp
+# WER on eval2000(tg)        12.3
+#           [looped:]        12.2
```

Comment:
Can you please do these WER comparisons using local/chain/compare_wer.sh?  Should be much easier, and then you can show what script you used to compare with.

Reply:
Sir, I mentioned the command used to compare WERs for SVD and Non-SVD versions of chain model. I don't know where the earlier person, who worked on this PR, left his model training folder backup. In order to derive the WERs for SVD retrained model on rt03 testset (mentioned in line 29), I took the SVD retrained model from Nagendra sir in order to prepare decoding graph and decode rt03.

But I have verified that the cmdline mentioned in the script comments works fine, using the training folders of my recent Non-ivector SVD retrained models on Noise-perturbed Multi-english data, which give slightly higher WERs than the ones mentioned in the script.